src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../Types/todoInterface';
+import { getUserById } from '../../services/user';
```

Comment:
Ensure that the `getUserById` function is correctly implemented in the `../../services/user` file. It should return a user object based on the provided `userId`.